test/planned_subvol_test.rb

```diff
@@ -207,4 +208,122 @@
     end
   end
 
+  context ".read_from_xml" do
```

Comment:
Don't get me wrong. I didn't ask you to cover the entire fallback list verbatim. I asked you to cover as many scenarios as possible with explanatory descriptions. Scenarios like having a duplicate subvolume in control.xml or all the possible combinations of negated and not negated architectures in the `arch` xml attribute.

I provided the tests at #200 as an example. I don't think those tests are more fragile than these ones. They don't cover the entire fallback list. They were, as a matter of fact, useful to find an existing bug and to fearlessly refactor one method. I would even like to believe they are useful to clarify something that is not completely clear (at least to me) in the documentation regarding negated architectures.